docs/workflow/debugging/coreclr/debugging.md

```diff
@@ -19,10 +19,16 @@ Debugging CoreCLR on Windows
 7. Set Command Arguments=`<managed app you wish to run>` (e.g. HelloWorld.dll)
 8. Set Working Directory=`$(SolutionDir)\..\..\..\bin\coreclr\Windows_NT.$(Platform).$(Configuration)`
     1. This points to the folder containing CoreCLR binaries.
-9. Press F11 to start debugging at wmain in corerun (or set a breakpoint in source and press F5 to run to it)
+9. Set Environment=`CORE_LIBRARIES=$(SolutionDir)\..\..\..\bin\pkg\<current tfm>\runtime\Windows_NT-$(Configuration)-$(Platform)`,
```

Comment:
I think it would be better to use the following path instead:
```suggestion
9. Set Environment=`CORE_LIBRARIES=$(SolutionDir)\..\..\..\bin\runtime\<current tfm>-Windows_NT-$(Configuration)-$(Platform)`,
```
It contains additional assemblies (many Microsoft.*, some other System.*, etc) which might be necessary for running some applications.